src/app/app.module.ts

```diff
@@ -5,7 +5,6 @@ import { BrowserModule } from '@angular/platform-browser';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
-
```

Comment:
Why was this line removed? Good practice to keep angular imports in separate section.